run-code-examples/order-sum-products/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Check how many products of a certain type have been ordered
+*/
+
+export default function main({order}) {
+  const titleToMatch = "test";
+
```

Comment:
Added this because it makes it easier to see something you should change, as a merchant